src/kemal.cr

```diff
@@ -17,13 +20,12 @@ at_exit do
   config.add_handler Kemal::Handler::INSTANCE
   config.add_handler HTTP::StaticFileHandler.new("./public")
 
-  server = HTTP::Server.new("0.0.0.0", config.port, config.handlers)
```

Comment:
Oh... But I pushed my code after pulling updates from your branch and I do remember that you bind it to `0.0.0.0` in one of your previous commits. What happened to that particular line?